cdn-server/cdn/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@wundergraph/cosmo-cdn",
```

Comment:
Do you think CDN is a good name for the package? It doesn't really express what it does. The module will upload persistent operations. What about `@wundergraph/cosmo-hono-po`

Reply:
We stick with the name.